docs/design-systems/templates.md

```diff
@@ -1,3 +1,112 @@
+  - - meta
+    - property: og:image
+      content: https://bluefox.email/assets/docs-share.png
+  - - meta
+    - property: og:url
+      content: https://bluefox.email/docs/projects
+  - - meta
+    - property: og:type
+      content: website
+  - - meta
+    - property: twitter:card
+      content: summary_large_image
+  - - meta
+    - property: twitter:title
+      content: Design System Templates | bluefox.email documentation
+  - - meta
+    - property: twitter:description
+      content: Discover how templates in the bluefox.email design system help you build structured, reusable email layouts. Combine blocks, define subjects, preview text, and set template behaviors to streamline email creation and project consistency.
+  - - meta
+    - property: twitter:image
+      content: https://bluefox.email/assets/docs-share.png
+---
+
+
 # Design system templates
 
-//TODO
+Templates are structured email designs that serve as a foundation for email campaigns, transactional messages, or triggered emails. Unlike blocks, which represent smaller reusable components, templates provide a complete email structure by combining multiple blocks along with additional elements such as buttons and variables.
+
+### **The importance of templates**:
+
+Templates play a crucial role in maintaining brand consistency, reducing design time, and ensuring email content aligns with predefined styles. They allow for efficient email creation by reusing blocks and applying custom variables, making them adaptable to different marketing or transactional needs.
+
+## Template categories
+
+Templates are grouped inside **Template categories**, which help organize them based on functionality or design purpose. Each category can contain multiple templates, making it easier to manage and locate relevant components.
+
+Users can:
+
+- **Create, update, or delete** template categories.
+- **Add templates** within a category.
+
+## Viewing templates
+
+Inside each template category, users will find a list of templates displayed as cards. Each template card includes:
+
+![A screenshot of the design system template card.](./design-system-template-card.webp)
+
+- **Template preview** – A quick look at the template's design.
+- **Template name** – The assigned name of the template.
+
+- **Edit name** – Clicking the pencil icon next to the name allows users to rename the template.
+
+- **Subject** – The subject line of the email, essential for email deliverability and engagement.
+
+- **Edit subject** – Clicking the pencil icon next to the subject allows users to update the subject.
+
+- **Preview text** – A short preview of the email content displayed in inboxes.
+
+- **Edit preview text** – Clicking the pencil icon next to the preview text allows users to update the preview text.
+
+- **On project creation** – A dropdown that defines the template’s role within a project. Options include:
```

Comment:
I would explain it a little bit more... so when they create the project, these templates will automatically added to the project... (as transactional, triggered or campaign.)